### DISCUSSION TITLE
hide scrollbar

### DISCUSSION THREAD

**Discussion by Shuvadip-Ghosh** [TomSchimansky/CustomTkinter#1602] [General]
I am using the customtkinter.CTkScrollableFrame and want to hide the scrollbar. How to do it.

**Comment by avalon60** [comment]
> I am using the customtkinter.CTkScrollableFrame and want to hide the scrollbar. How to do it.

If you don't want the scrollbar, why don't you just use CTkFrame?

If you need to show/hide the scrollbar, it doesn't look like there is a method supplied. Looking at the code, you could possibly do something like CTkScrollableFrame._scrollbar.grid_remove() and to redisplay, use CTkScrollableFrame._scrollbar.grid(). 

However this is considered bad practice, and  you should be aware, that any class, or module object beginning with an underscore, is considered private and shouldn't really be referenced outside of its module.  I've seen others doing this, and it's  questionable and can cause code to be less reliable, across versions. When leading with an underscore, the developer is essentially saying, keep away, this is internal and may be subject to change.

**Comment by Akascape** [comment]
@Shuvadip-Ghosh 
Just hide it by matching the color of frame.

 ```python
# get the default colors of frame
frame_fg = customtkinter.ThemeManager.theme["CTkFrame"]["fg_color"] 
```      
Add these arguments in the CTkScrollableframe:
```python     
...
scrollbar_button_hover_color=frame_fg,
scrollbar_button_color=frame_fg,
...
```

**Comment by giano574** [comment]
It makes sense to be able to hide the scrollbar when the content doesn't overflow the CTkScrollableFrame's bounds. I would like such a feature
